react/tic-tac-toe/src/app/screens/Signup/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../../scss/colors.scss";
+
+.button {
+  max-width: 252px;
+  width: 100%;
+  height: 32px;
+  border: 2px solid $wolox-green;
+  border-radius: 5px;
+  margin-bottom: 24px;
```

Comment:
There is no need to change it now, since it's just a minor detail. But, it's good to keep it in mind: In wolox style guide we order css properties alphabetically